src/pages/import/index.tsx

```diff
@@ -87,7 +87,7 @@ export function ImportTimetable(): JSX.Element {
           {!loading && data ? calendarView ? calendarComponent : listView : <Spinner size="xl" />}
         </VStack>
       ) : (
-        <VStack pt={2} w="100%" height="100%" overflow="hidden" grow={1}>
+        <VStack pt={2} w="100%" height="100%" overflow="hidden" flexGrow={1}>
```

Comment:
is there a reason for prefixing these props with `flex`?

Reply:
i thought chakra added these  "sub-`flex`" props to flex components to simplify the prop names

Reply:
Updating removed those props so I had to change them.